ReactWindows/ReactNative.Shared/Modules/I18N/I18NUtil.cs

```diff
@@ -32,11 +39,21 @@ public static bool IsRightToLeftAllowed
         {
             get
             {
+#if WINDOWS_UWP
                 return (bool?)ApplicationData.Current.LocalSettings.Values[AllowRTL] ?? false;
+#else
+                var result = default(bool);
+                var parsed = bool.TryParse(ConfigurationManager.AppSettings[AllowRTL], out result);
+                return (bool)(object)(parsed && result);
+#endif
```

Comment:
I'd rather make it larger and have the #ifs isolated to a smaller number of locations.

---
In reply to: [85239327](https://github.com/ReactWindows/react-native-windows/pull/831#discussion_r85239327) [](ancestors = 85239327)